File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
       
       Set<Integer> two =new HashSet<Integer>();
        for (int i = 0; i < nums.length; i++) {
            if (two.contains(nums[i])) {
                return true;
            } else {
                two.add(nums[i]);
            }

        }
        return false;

  
    }
}

File: 217-contains-duplicate/contains-duplicate.ts
function containsDuplicate(nums: number[]): boolean {
    nums.sort()
    let num1=[]
    for(let i=0;i<nums.length;i++){
       if(nums[i]==nums[i-1]){
        return true
       }else{
        continue
       }
    }
    return false

};